game/static/game/css/landing.css

```
@@ -79,10 +92,14 @@ body {
     align-items: center;
     padding: 1.5rem 5%;
     background-color: var(--nav-bg);
+
     border-bottom: 1px solid var(--border-color);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)